knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+
+## Host
+
+The hostname and the port of the server to which the request was sent.
+
+## URL
+
+The URL path, file, and query string from the request.
+
+## Method
+
+The HTTP method used by the request (__GET__, __POST__, etc.). The HTTP methods are used to set the desired action that needs to be applied to a given resource.
+
+- __GET__: The GET method requests a representation of the specified resource. Requests using GET should only retrieve data.
+- __HEAD__: The HEAD method asks for a response identical to that of a GET request, but without the response body.
+- __POST__: The POST method is used to submit an entity to the specified resource, often causing a change in state or side effects on the server.
+- __PUT__: The PUT method replaces all current representations of the target resource with the request payload.
+- __DELETE__: The DELETE method deletes the specified resource.
+- __CONNECT__: The CONNECT method establishes a tunnel to the server identified by the target resource.
+- __OPTIONS__: The OPTIONS method is used to describe the communication options for the target resource.
+- __TRACE__: The TRACE method performs a message loop-back test along the path to the target resource.
+- __PATCH__: The PATCH method is used to apply partial modifications to a resource.
+
+## Body
+
+Shows the number of bytes in the response body.
+
+## Caching
+
+Values (if any) from __Cache-Control__ and __Expires__ headers. In case, where both headers are present, the column will display first the values from __Cache-Control__ and then the values for __Expires__ header (separated by semicolon).
+
+The __Cache-Control__ header is containing instructions for caching (for both requests and responses). An example of standard __Cache-Control__ header:
+
+```
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+```
+
+The __Expires__ header contains the date/time after which the response is considered invalid. When the __Expires__ header has invalid date value (for example, __0__ or __-1__) that means that the expiration date is in the past, and the resource is already considered expired. An example for standard __Expires__ header:
+
+```
+Expires: Fri, 20 Apr 2020 19:00:00 GMT
+```
+
+>tip In a scerario where a __Cache-Control__ header with __max-age__ or __s-maxage__ is set, the value from __Expires__ will be ignored.
+
+## Content-Type
+
+The __Content-Type__ header is the header which is originated from the response.
```

Comment:
"is originated from" might be correct but it sounds a bit strange, at least to me - perhaps **originated** should be **originating**? 
You can also consider removing the first "header":
The __Content-Type__ is a header which is originating from the response.